src/libraries/System.Net.Security/src/System/Net/Security/SslAuthenticationOptions.cs

```diff
@@ -11,6 +11,7 @@ namespace System.Net.Security
 {
     internal sealed class SslAuthenticationOptions
     {
+        private const string EnableOcspStaplingContextSwitchName = "System.Net.Security.EnableOcspStaplingOnServerCertificate";
```

Comment:
I worry that the name is wrong in two ways:

1) OCSP is enabled already, if you provide an SslStreamCertificateContext instead of just a certificate.
2) This setting only matters on Linux.

So explicitly setting it to false doesn't turn off (1), and has no effect on Windows/macOS/iOS/Android, but sounds like it does.  And it's on in a bajunch of cases without being set to true.

So, really, it's more like "EnableServerOcspStaplingWithoutContext", or "EnableServerOcspStaplingFromOnlyCertificate" or something.

Reply:
That is the exact reason, I wanted to mark this PR as ready for review because I wasn't also sure about the naming and was looking for some feedback.

Tbh, `EnableServerOcspStaplingFromOnlyCertificateOnLinux` sounds good to me, but I'm not sure if there is any pushback for that.

@wfurt @rzikm any pushback?

Reply:
Either of the two suggestions is fine with me